modules/KalturaSupport/resources/uiConfComponents/playbackRateSelector.js

```diff
@@ -51,6 +51,7 @@
 				this.speedSet[i] = Math.abs( this.speedSet[i] );
 			}
 			this.addBindings();
+            this.getPlayer().triggerHelper( 'updatedPlaybackRate', this.currentSpeed); // trigger report to Analytics
```

Comment:
`updatedPlaybackRate` event means that the speed was changed from the plugin, which a user action, so not sure if adding the event like this is the correct way to go.
Probably need to add API to get rate on the base player and query it on analytics plugin CTOR